iotedged/src/main.rs

```diff
@@ -382,11 +382,11 @@ struct Options {
 	device_id: String,
 
 	/// `keyd` API endpoint.
-	#[structopt(long, default_value = "unix:///var/lib/aziot/keyd.sock")]
+	#[structopt(long, default_value = "unix:///run/aziot/keyd.sock")]
```

Comment:
It seems unlikely because /run is `rwx` to `root` user only. But, what's the behavior if `/run/aziot` already exists? I assume the service will fail to start?

Reply:
For socket-activation, the directory and the .sock itself will have been created by systemd, as I wrote in the other comment.

For non-socket-activation, the user has to ensure it's been created to be `rwx` to the three users, and `r-x` to the three groups. The socket files themselves will have the correct permissions when the services bind them, because they'll follow the user and group of the service, which is set in the service file (the group isn't, but the primary group of the user was set at user-creation time).